packages/wp-source/src/libraries/response-utils.ts

```diff
@@ -0,0 +1,7 @@
+export const getTotal = (response: Response): number =>
+  parseInt(response.headers.get("X-WP-Total"));
+
+export const getTotalPages = (response: Response): number =>
+  parseInt(response.headers.get("X-WP-TotalPages"));
```

Comment:
What happens here if the X-WP-TotalPages or X-WP-Total is not present? Sometimes people delete them in Nginx.

Reply:
Uhm, if that is the case, I don't know from which place that values could be obtained.

Taxonomies has a `count` property that gives you the `total`value but not `totalPages`, and authors and other archives don't even have that property.

We could maybe use the `_envelope` param in the `api.get` requests and receive all response data in the body, including headers and status code, but that would change the return value of that method...

@luisherranz, any idea?

Reply:
I forgot to link the [docs](https://developer.wordpress.org/rest-api/using-the-rest-api/global-parameters/#_envelope) of the `_envelope` property.

Reply:
I wasn't asking how to solve but what happens when those headers are not present. `parseInt(undefined);` and `parseInt(null);` are `NaN`. It is something we can expect and it shouldn't break the app.

Reply:
Oh, I didn't know about the `_envelope` query. Interesting :)

Reply:
Maybe we can make sure nothing is broken if those headers are not present right now and if people report having problems with this in the future switch to use the `_envelope` query param.

Reply:
Ok, I'll check everything works without those headers.

Reply:
Thanks David.